### PR TITLE
Socint 334

### PR DESCRIPTION
**Pull request by philwhiles**
EQ Launch functionality updated

### CODE REVIEW

File: src/test/resources/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/PackageFixture.Uac.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+[
+  {
+		"id": "dc567365-2812-4bdd-939f-ca9045bb02d2",
```
Comment (PeterBochel) [inline src/test/resources/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/PackageFixture.Uac.json:3]: The indentation is inconsistent (tabs vs. spaces).
Also same for previous json file

File: src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/model/CollectionExercise.java [hunk 2]
```diff
@@ -42,4 +44,36 @@
...
+    long waveCutOffDays = strict ? waveLength : periodBetweenWaves;
+    int wave = -1;
+    for (int n = 0; n < numberOfWaves; n++) {
+      LocalDateTime waveStart = startDate.plusDays(n * periodBetweenWaves);
+      if (date.isAfter(waveStart) && date.isBefore(waveStart.plusDays(waveCutOffDays))) {
+        wave = n;
+        break;
+      }
+    }
+    return wave == -1 ? Optional.empty() : Optional.of(wave);
```
Comment (PeterBochel) [inline src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/model/CollectionExercise.java:77]: I suspect we'll be re-visiting this. But as previously discussed, more than good enough for now.

**Review by PeterBochel — APPROVED** [review]
Approving, with comment about json formatting

File: src/test/resources/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/PackageFixture.Uac.json (hunk 1 above)
Comment (philwhiles) [reply src/test/resources/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/PackageFixture.Uac.json]: fixed

File: src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/model/CollectionExercise.java (hunk 2 above)
Comment (philwhiles) [reply src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/model/CollectionExercise.java:77]: agreed

**Pull request merged**